packages/sdk/src/conversation/interface.ts

```diff
@@ -261,6 +309,95 @@ export interface TeamsFxAdaptiveCardActionHandler {
   handleActionInvoked(context: TurnContext, actionData: any): Promise<InvokeResponse>;
 }
 
+/**
+ * Interface for SSO configuration for BotSSO
+ */
+export interface SsoConfig {
+  /**
+   * Custom sso execution activity handler class which should implement the interface {@link SsoExecutionActivityHandler}. If not provided, it will use {@link DefaultSsoExecutionActivityHandler} by default
+   */
+  CustomSsoExecutionActivityHandler?: new (ssoConfig: SsoConfig) => SsoExecutionActivityHandler;
+
+  /**
+   * The list of scopes for which the token will have access, if not provided, it will use graph permission ["User.Read"] by default
+   */
+  scopes?: string[];
+
+  /**
+   * Conversation state for sso command bot, if not provided, it will use internal memory storage to create a new one.
+   */
+  conversationState?: ConversationState;
```

Comment:
is it possible to just provide a Storage and we help user create conversationState, userState? By the way, can we reuse same storage for xxxState and dedupStorage? If yes, we may also remove dedupStorage config

Reply:
Not sure if there are some cases that user want to use different storage for conversationState, userState, so I just expose these 3 properties for flexibility.